src/main/resources/windup-realm/windup-realm.json

```diff
@@ -779,7 +779,7 @@
     {
       "id": "739a78cd-ab8d-427a-93f7-4af38f0eab31",
       "clientId": "mta-web",
-      "name": "Migration Toolkit for Applications Web Console",
+      "name": "Tackle Analysis Web Console",
```

Comment:
Just like above.

Reply:
Thanks @mrizzi . You're right no branding changes should be done.